fire/cli/ts/__init__.py

```diff
@@ -9,4 +26,130 @@ def ts():
     pass
 
 
+def _print_tidsserieoversigt(tidsserieklasse: type, punkt: Punkt = None):
+    """
+    Oversigt over tidsserier af en bestemt types
+
+    raises:     SystemExit
+    """
+
+    def identgnss(punkt: Punkt):
+        return punkt.gnss_navn
+
+    def identident(punkt: Punkt):
+        return punkt.ident
+
+    if tidsserieklasse == GNSSTidsserie:
+        foretrukken_ident = identgnss
+    else:
+        foretrukken_ident = identident
+
+    if punkt:
+        tidsserier = [ts for ts in punkt.tidsserier if isinstance(ts, tidsserieklasse)]
+    else:
+        tidsserier = (
+            fire.cli.firedb.session.query(tidsserieklasse)
+            .filter(tidsserieklasse._registreringtil == None)
+            .all()
+        )  # NOQA
+
+    if not tidsserier:
+        raise SystemExit("Fandt ingen tidsserier")
+
+    tabel = Table("Ident", "Tidsserie ID", "Referenceramme", box=box.SIMPLE)
+
+    # Sorter tidsserier efter punkt
+    tidsserier.sort(key=lambda ts: (foretrukken_ident(ts.punkt)))
+
+    tidsserier = tidsserier[:100]
```

Comment:
Jeg går ud fra du begrænser til 100 her for at undgå at outputtet bliver uoverskueligt? 

En alternativ løsning kan være at spytte outputtet ud via en pager, fx:

```python
...
console = Console()
with console.pager():
    console.print(tabel)
```

Hvis tabellen er for lang til at være på skærmen får man mulighed for at bladre op og ned i den, ligesom hvis man piper en lang fil over i `less`.

Reply:
Ja det bliver uoverskueligt og tager lang tid. Lige nu er der 3800 højdetidsserier på test-databasen, og det tager lidt tid at få dem spyttet ud. Kendte ikke til "pager", det virker meget smart. 
Men stadig... det tager en evighed at få alle tidsserierne scrollet igennem hvad enten man får dem vist lidt ad gangen med pager eller får vist alt sammen på én gang. Det giver ikke meget værdi, så måske skal man bare fjerne muligheden helt for at få vist ALLE højdetidsserierne?

Reply:
> Det giver ikke meget værdi, så måske skal man bare fjerne muligheden helt for at få vist ALLE højdetidsserierne?

Ja, det synes jeg. Det kan godt give mening med en liste visning hvis man kan begrænse antallet af tidsserier der vises frem, fx ved at indsnævre feltet ud fra nogle kriterier bruger stiller op. Jeg er ikke kommet hele vejen igennem koden endnu, så jeg ved ikke om det allerede er muligt :)

Reply:
Det er muligt at få vist alle tidsserier, alle ts'er tilhørende et bestemt punkt, eller få vist koordinater til en navngiven tidsserier. Men det er ikke muligt at begrænse antallet som sådan. `ts hts` er funktionelt identisk med den allerede eksisterende `ts gnss`. Eneste forskel er, at de håndterer højde- hhv. gnss tidsserier.

Reply:
Fjern muligheden for at få vist den store samlede list, og så kan vi på et senere tidspunkt lave mulighed for at filtrere på fx navnet af tidsserien (opret gerne et issue på det).

Reply:
har sat click-argumentet til `required=True` så man nu ikke kan lave søgning uden input. har lavet et issue #780 på mulighed for at filtrere søgeresultater